docs/routines/posthog-questions.md

```
@@ -581,41 +581,71 @@ Public-safe-default. Aggregates only — never names a single user.
+  WHERE step IS NOT NULL
   GROUP BY step
-  HAVING step IS NOT NULL
   ORDER BY failure_pct DESC
   ```
 - **Bind variables**: `{ window: "7d" }` (default; `30d` for monthly digest, `1d` for hot triage).
-- **Output schema**: every field public.
-- **Required emit-side events**: the four `*_started/_success/_failed` event triplets above (all present in `packages/shared/src/modules/app/analytics-events.ts`). Note that `work_approval_success` may live primarily on the Admin project (262122) — when this question runs against the App project, `work_approval` will report 100% failure even when admin successes exist. Run the query separately against project 262122 for the admin-side success counts and merge in the routine.
-- **Anomaly thresholds (consumer)**: failure rate > 50% AND absolute failed count ≥ 5 over the window → file a Linear anomaly Issue. 100% failure with absolute count ≥ 5 → P2/urgent. The thresholds catch real product breakage rather than tiny-N noise.
+- **Output schema**: every field public. `failing_persons` is a distinct count, never an identifier — it stays public-safe.
+- **Cancellation-detection coverage** (verified against 120d of App data, 2026-08-10). Detection is only as good as the emit side, so treat `cancelled_count` as a floor, not a complete count:
+  | Step | Signal | Reliable? |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)